gitbook-docs/market/overview.md

```diff
@@ -17,24 +17,24 @@ to buy and sell future values of Bitcoin via perpetual contracts.
 These are called derivative markets, as you do not own the underlying instrument
 but are just speculating on its price.
 
-`\=nil;` Proof Market belongs to the class of spot markets and is designed
+`=nil;` Proof Market belongs to the class of spot markets and is designed
 as a decentralized and distributed system.
 
 ## Circuits and statements
 
 Markets tend to arrive at a fair price through interactions between buyers and sellers.
-In the Proof Market, the object of trade are statements — circuits 
+In Proof Market, the object of trade are statements — circuits 
 prepared for publishing on the market.
 Circuits are algebraic representations of an algorithm, created by the proof system
 and compiled with the zkLLVM compiler.
 For example, it can be MINA or Solana state-proofs.
-To publish a circuit to the Proof Market, you should prepare a statement and attach
+To publish a circuit to Proof Market, you should prepare a statement and attach
```

Comment:
Still not quite true: one doesn't attach metatdata to a statement. Statements just contain metadata and a circuit. We could say that user builds a statement by attaching metadata to a circuit.

Reply:
Fixed?

Reply:
yes, now it's better